doc-content/enterprise-only/installation/jws-zip-install-proc.adoc

```diff
@@ -1,62 +1,50 @@
-.. Unzip the `{PRODUCT_FILE}-add-ons.zip` file. The 
-ifdef::PAM[]
-`{PRODUCT_INIT}-{PRODUCT_VERSION}-{URL_COMPONENT_KIE_SERVER}-jws.zip`
-endif::PAM[]
-ifdef::DM[]
-`{PRODUCT_INIT}-{PRODUCT_VERSION}-{URL_COMPONENT_KIE_SERVER}-jws.zip`
-endif::DM[]
- file is in the unzipped directory.
---
 
 .Procedure
+. Download the `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip` file:
+.. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product:* {PRODUCT_SHORT}
+* *Version:* {PRODUCT_VERSION}
+
+.. Click *Download* next to *{PRODUCT} {PRODUCT_VERSION_LONG} Add Ons* (`{PRODUCT_FILE}-add-ons.zip`).
+.. Unzip the `{PRODUCT_FILE}-add-ons.zip` file. The
+`{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
+ file is in the unzipped directory.
 . Extract the 
 ifdef::PAM[]
-`{PRODUCT_INIT}-{PRODUCT_VERSION}-{URL_COMPONENT_KIE_SERVER}-jws.zip`
+`{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
 endif::PAM[]
 ifdef::DM[]
-`{PRODUCT_INIT}-{PRODUCT_VERSION}-{URL_COMPONENT_KIE_SERVER}-jws.zip`
+`{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
 endif::DM[]
- archive to a temporary directory. In the following examples this directory is called `__TEMP_DIR__`.
+ archive to a temporary directory. In the following examples, this directory is called `__TEMP_DIR__`.
 . Copy the 
 ifdef::PAM[]
-`__TEMP_DIR__/{PRODUCT_INIT}-{PRODUCT_VERSION}-{URL_COMPONENT_KIE_SERVER}-jws/kie-server.war`
+`__TEMP_DIR__/{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws/kie-server.war`
 endif::PAM[]
 ifdef::DM[]
-`__TEMP_DIR__/{PRODUCT_INIT}-{PRODUCT_VERSION}-{URL_COMPONENT_KIE_SERVER}-jws/kie-server.war`
+`__TEMP_DIR__/{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws/kie-server.war`
 endif::DM[]
  directory to the `_JWS_HOME_/tomcat/webapps` directory.
 +
-WARNING: Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the Red Hat JBoss Web Server instance.
+WARNING: Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the {JWS} instance.
 . Remove the `.war` extensions from the `kie-server.war` folder.
 . Open the `_JWS_HOME_/tomcat/conf/tomcat-users.xml` file in a text editor.
-. Add users and roles to the `_JWS_HOME_/tomcat/conf/tomcat-users.xml` file. In the following example, `<ROLE_NAME>` is a role supported by {PRODUCT}. 
-//For a list of supported roles, see <<dm-roles-con>>.  
-`<USER_NAME>` and `<USER_PWD>` is a user and password combination of your choice:
+. Add users and roles to the `_JWS_HOME_/tomcat/conf/tomcat-users.xml` file. In the following example, `<ROLE_NAME>` is a role supported by {PRODUCT} and `<USER_NAME>` and `<USER_PWD>` is a user and password combination of your choice:
```

Comment:
XML is a hierarchical format. I think you need to tell the user under what tag to add these tags.

Reply:
Actually, in this file there aren't headings, its a short file, and there are place holder tags that are obvious to edit.